src/DssSpell.sol

```diff
@@ -59,187 +51,224 @@ contract DssSpellAction is DssAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
     // uint256 internal constant X_PCT_RATE = ;
-
-    // ---------- Math ----------
-    uint256 internal constant WAD = 10 ** 18;
-    uint256 internal constant RAD = 10 ** 45;
+    uint256 internal constant FIVE_PT_FIVE_PERCENT_RATE   = 1000000001697766583380253701;
+    uint256 internal constant SIX_PT_FIVE_PERCENT_RATE    = 1000000001996917783620820123;
```

Comment:
This doesn't follow the naming convention. Can we update it to `SIX_PT_FIVE_PCT_RATE`?